src/registry.rs

```diff
@@ -265,4 +280,119 @@ mod tests {
             panic!("Should be a composite type definition")
         }
     }
+
+    #[test]
+    fn clear_docs() {
+        #[allow(unused)]
+        /// docs
+        struct S {
+            /// docs
+            pub t: bool,
+            pub u: u8,
+        }
+
+        impl TypeInfo for S {
+            type Identity = Self;
+
+            fn type_info() -> Type {
+                Type::builder()
+                    .path(Path::new("S", module_path!()))
+                    .docs(&[" docs"])
```

Comment:
This leading space is a bit annoying. We could take an opinionated stance and strip off leading ` ` or `\t`.

Reply:
Changed it to remove the leading space, maybe not the best solution but it works. Leaving tabs because they might be for formatting.